packages/synapse-react-client/src/components/DataGrid/DataGridTypes.ts

```diff
@@ -10,11 +31,8 @@ export interface QueryInput {
   input: string
 }
 
-export interface ModelSnapshot {
-  columnNames: string[]
-  columnOrder: number[]
-  rows: string[]
-}
+export type GridModel = ReturnType<typeof Model.create<typeof gridSchema>>
```

Comment:
```suggestion
export type GridModel = ReturnType<typeof Model.create<typeof _gridSchema>>
```
match the renamed var in the suggestion above